primary.tf

```diff
@@ -0,0 +1,68 @@
+resource "aws_s3_bucket" "primary" {
+  provider = aws.primary
+
+  bucket = var.primary_name
+
+  force_destroy = var.force_destroy
+
+  tags = var.tags
+}
+
+resource "aws_s3_bucket_versioning" "primary" {
+  provider = aws.primary
+
+  bucket = aws_s3_bucket.primary.bucket
+
+  versioning_configuration {
+    status = "Enabled"
+  }
+}
+
+resource "aws_s3_bucket_replication_configuration" "primary" {
+  provider = aws.primary
+
+  depends_on = [aws_s3_bucket_versioning.primary]
+
+  role   = aws_iam_role.replication.arn
+  bucket = aws_s3_bucket.primary.bucket
+
+
+  rule {
+    id = aws_s3_bucket.secondary.bucket
+
+    filter {}
```

Comment:
let's mention in a comment what this empty block means.

Reply:
see 40c93749c9dbd7fd158bdd06f0740b5aaee5bc33